src/bundle/MarkdownEditorView.tsx

```diff
@@ -295,6 +297,9 @@ const MarkupSearchAnchor: React.FC<MarkupSearchAnchorProps> = ({mode}) => (
 function Settings(props: EditorSettingsProps & {stickyToolbar: boolean}) {
     const wrapperRef = useRef<HTMLDivElement>(null);
     const isSticky = useSticky(wrapperRef) && props.toolbarVisibility && props.stickyToolbar;
+
+    if (!props.renderPreviewButton && !props.settingsVisible) return null;
```

Comment:
This looks like breaking changes, I suggest passing the default values so that it works for previous cases when renderPreviewButton and settingsVisible are not set

Reply:
I replaced the highlighted code fragment, tested it with the main branch (behavior with preview and settings turned off, as well as without props), the behavior was expected